### ISSUE TITLE
Agent browser in Grid

### ISSUE THREAD

**Issue by pcaillou** [gama-platform/gama#1237]
```
What steps will reproduce the problem?
1.Open any model that uses a grid
2.Try to inspect the cell population using Agents->Population of cell -> Browser population


What is the expected output? What do you see instead?
The browser is empty

Please use labels and text to provide additional information.
svn version & mac os x

```

Original issue reported on code.google.com by `agrignard` on 2015-05-28 02:10:24

**Comment by AlexisDrogoul** [comment]
The population browser is still completely broken. Except to see a fix when it will be rewritten.

**Comment by AlexisDrogoul** [comment]
Fixed in 218d4a37f8911a286fb7520e810a9b64562af23d
